mk/spksrc.service.mk

```diff
@@ -182,6 +170,13 @@ ifneq ($(strip $(SERVICE_WIZARD_SHARE)),)
 		'."data-share" = {"shares": [{"name": $$share, "permission":{"rw":[$$user]}} ] }' $@ 1<>$@
```

Comment:
Something interesting to note:
> Since 7.0-41201, package center will create a symlink under `/var/packages/[package_id]/shares/` named by share folder pointing to share folder path.

Reply:
I have to test shared folders with DSM6 anyway. I will look whether dsm6 resource worker will create such links too.